python/lsst/afw/image/filter.cc

```diff
@@ -20,26 +20,86 @@
  * see <https://www.lsstcorp.org/LegalNotices/>.
  */
 
+#include <memory>
+
 #include <pybind11/pybind11.h>
-//#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/stl.h>
+
+#include "lsst/daf/base/PropertySet.h"
+#include "lsst/pex/policy/Policy.h"
+#include "lsst/afw/image/Filter.h"
 
 namespace py = pybind11;
+using namespace pybind11::literals;
 
-using namespace lsst::afw::image;
+namespace lsst {
+namespace afw {
+namespace image {
 
 PYBIND11_PLUGIN(_filter) {
     py::module mod("_filter", "Python wrapper for afw _filter library");
 
     /* Module level */
+    py::class_<FilterProperty, std::shared_ptr<FilterProperty>> clsFilterProperty(mod, "FilterProperty");
+    py::class_<Filter> clsFilter(mod, "Filter");
+
+    mod.def("stripFilterKeywords", &detail::stripFilterKeywords, "metadata"_a);
```

Comment:
You are right. However, that is how our SWIG did it, so it was required for backwards compatibility.